src/main/java/bio/terra/cli/command/Group.java

```diff
@@ -0,0 +1,28 @@
+package bio.terra.cli.command;
```

Comment:
I'm surprised git detected a delete & replace rather than a rename here. The similarity is really high. If you really want to save the history, I think there's a way to adjust the similarity threshold. Probably not worth it here.

Reply:
Yeah I don't know why it did that either. I'm going to leave it as is for now, but good to know we can adjust it in the future if this keeps happening.